packages/amplify_auth_cognito/example/lib/tempDelete.dart

```diff
@@ -0,0 +1,43 @@
+class Model<T> {
```

Comment:
what is this file about?

Reply:
mistake, thanks removed in merged branch